src/components/calcite-pick-list/shared-list-logic.ts

```diff
@@ -30,6 +31,7 @@ export const sharedListMethods = {
   initialize(this: CalcitePickList | CalciteValueList): void {
     this.setUpItems();
     this.setUpFilter();
+    this.emitCalciteListChangeEventDebounced = debounce(this.emitCalciteListChangeEvent, 0);
```

Comment:
Nitpick: Do we need to expose both the debounced and non-debounced method? Could we do something like this?

***disclaimer: untested code ⚠️***

```tsx
// shared-list-logic.ts
initialize(this: CalcitePickList | CalciteValueList): void {
  // ...
  this.emitCalciteListChange = debounce(this._internalCalciteListChangeEvent, 0);
}

_internalCalciteListChangeEvent(this: CalcitePickList | CalciteValueList): void {
  // previously `emitCalciteListChange` from pick-list and value-list
  this.calciteListChange.emit(this.selectedValues);
}
```

```tsx
// pick-list & value-list
emitCalciteListChangeEvent: () => void;

// no more 👇
// emitCalciteListChangeEvent(): void {
//   this.calciteListChange.emit(this.selectedValues);
// }
```

Reply:
noice!